test.js

```diff
@@ -1,82 +1,71 @@
-const tape = require('tape')
-const Transmission = require('./index')
 const os = require('os')
-const path = require('path')
 const fs = require('fs')
+const path = require('path')
+const assert = require('node:assert/strict')
+const { describe, before, beforeEach, afterEach, it } = require('node:test')
+
+const Transmission = require('./index')
 
 const tmpDir = path.join(os.tmpdir(), 'transmission-native')
+const APP_NAME = 'transmission'
```

Comment:
A practice widely used in the node ecosystem is to prefix a constant name with `k,` and follow the camel case.

Not a requirement from my side, just a proposal.

```suggestion
const kAppName = 'transmission'
```

Examples: 
- https://github.com/nodejs/node/blob/1f75a9513fc829fbd5326a5c9bfa7678b7431eb8/lib/internal/console/constructor.js#L83-L95
- https://github.com/fastify/fastify/blob/1e9c10fc538abf13c1d9389daf40c220c51b4ec6/lib/validation.js#L3-L8

Reply:
I dit not know this was a thing actually. This seems to be the [Hungarian_notation](https://en.wikipedia.org/wiki/Hungarian_notation) ? I would say this does not seem to be a widely used convention as far as I know, at least in Javascript projects, so we might not adopt this convention at least for now ?